File: Programmers/JAVA/lv1/lv1_제일 작은 수 제거하기.java
https://programmers.co.kr/learn/courses/30/lessons/12935

class Solution {
    public int[] solution(int[] arr) {
        if(arr.length==1){
            int[] answer = {-1};
            return answer;
        }
        
        int min = arr[0];
        for(int i = 1; i < arr.length; i++){
            min = Math.min(min, arr[i]);
        }
        
        int answer[] = new int[arr.length-1];
        int count = 0;
        for(int j = 0; j < arr.length; j++){
            if(arr[j] != min){
                answer[count]=arr[j];
                count++;
            }
            else
                continue;
        }
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_제일 작은 수 제거하기.cs
using System.Collections.Generic;

public class Solution
{
    public int[] solution(int[] arr)
    {
        int[] answer = new int[] { };

        if (arr.Length == 1)
        {
            answer = new int[] { -1 };
            return answer;
        }

        List<int> list = new List<int>();
        int min = arr[0];
        foreach (int i in arr)
        {
            list.Add(i);
            if (min >= i)
                min = i;
        }
        list.Remove(min);
        answer = list.ToArray();

        return answer;
    }
}